packages/android-dark-mode-support/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  input: 'dist/esm/index.js',
+  output: [
+    {
+      file: 'dist/plugin.js',
+      format: 'iife',
+      name: 'capacitorTorch',
```

Comment:
iife name is wrong, I guess it should be `capacitorAndroidDarkModeSupport`

Reply:
Good catch, thanks!